07LessonSeven/in-class-project/index.html

```diff
@@ -1,10 +1,55 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-    <title>Template</title>
-</head>
-
-<body></body>
-
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="css/my-style.css">
+    <title>NASA</title>
+
+    <header>
+    <style>
+      h1 {
+        padding: 200px;
```

Comment:
no style tag here please! this should be deleted and make the style changes in the CSS